### ISSUE TITLE
valid-describe blows up on template strings (backticks)

### ISSUE THREAD

**Issue by nevir** [jest-community/eslint-plugin-jest#75]
I have a pattern where I prefer backticks for test/describe names (makes apostrophes easier to juggle, and helps call out strings that are user-visible vs those that aren't)

However, `valid-describe` blows up on 'em:

```js
describe(`foo`, () => { … }); // Error: First argument must be name
describe('foo', () => { … }); // OK
```

**Comment by macklinu** [comment]
Thanks for the bug report - I'll open a PR to address this case in `valid-describe`. 👍

**Comment by SimenB** [comment]
@nevir The fix should be available within 5 minutes 🙂

**Comment by nevir** [comment]
Nice turnaround time!
